src/preallocate.jl

```diff
@@ -450,19 +450,19 @@ end
 
 ###################################################################
 
-struct DiagnosticVars{T,Tprog}
-    RungeKutta::RungeKuttaVars{Tprog}
-    Tendencies::TendencyVars{Tprog}
-    VolumeFluxes::VolumeFluxVars{T}
-    Vorticity::VorticityVars{T}
-    Bernoulli::BernoulliVars{T}
-    Bottomdrag::BottomdragVars{T}
-    ArakawaHsu::ArakawaHsuVars{T}
-    Laplace::LaplaceVars{T}
-    Smagorinsky::SmagorinskyVars{T}
-    SemiLagrange::SemiLagrangeVars{T}
-    PrognosticVarsRHS::PrognosticVars{T}        # low precision version
-end
+# struct DiagnosticVars{T,Tprog}
+#     RungeKutta::RungeKuttaVars{Tprog}
+#     Tendencies::TendencyVars{Tprog}
+#     VolumeFluxes::VolumeFluxVars{T}
+#     Vorticity::VorticityVars{T}
+#     Bernoulli::BernoulliVars{T}
+#     Bottomdrag::BottomdragVars{T}
+#     ArakawaHsu::ArakawaHsuVars{T}
+#     Laplace::LaplaceVars{T}
+#     Smagorinsky::SmagorinskyVars{T}
+#     SemiLagrange::SemiLagrangeVars{T}
+#     PrognosticVarsRHS::PrognosticVars{T}        # low precision version
+# end
```

Comment:
I see you moved them into `src/model_setup.jl` that's fine but then I'd actually delete them here not just uncomment. I only leave commented code in when to document that some functionality could be added here at some point in the future or to explain with code what's going on